CVLByExample/summary/with-env/WithEnvCVLFunctionSummary/README.md

```diff
@@ -0,0 +1,9 @@
+# Function Summarization with env
+
+## test.spec
+Includes a function summary with the flag "with env".
+
+Run this spec via
+```certoraRun runWithEnv.conf```
+
+[The report of this run](https://prover.certora.com/output/1902/48328b0f6fb84b2389a6f00b26f64a91?anonymousKey=3f1c5f63fe2d30a187ab0553ce761fcdcf27a1dd)
```

Comment:
maybe movethis readme one level up and all explain withEnvGhost folder

Reply:
I moved it up but the ghost example is waiting for this pr
https://certora.atlassian.net/browse/CERT-3972